### ISSUE TITLE
Mark notes as important

### ISSUE THREAD

**Issue by OpenGreenStreet** [openstreetmap/openstreetmap-website#3626]
### Description

I would like to suggest that notes can be marked as important (high priority) in exceptional cases.

For example: a road has been closed due to construction. In my view, this is important information that should be added to OSM as soon as possible.

Background: 
I contribute to OSM mostly via StreetComplete. When things catch my eye, I leave a note. I then hope for someone with experience to implement it.

In my experience, a note can be edited very quickly but can also remain unedited for several months. (By then, the construction work may already be finished).

See also: https://github.com/streetcomplete/StreetComplete/issues/4237#issuecomment-1195414118

**Comment by tomhughes** [comment]
Who gets to decide if a note is important though? The reporter? But all reporters naturally think what they are reporting is important...

It's not like marking a note as "important" is going to magically persuade somebody to look at it - we don't have an army of paid employees that we can direct to prioritise certain things. OpenStreetMap is a do-ocracy and no amount  of marking things as important is going to have a significant attempt on when something gets fixed.

I don't even agree that road closures should generally be added to OSM anyway accept for the rare cases where they are going to last for more than a week or so.

**Comment by OpenGreenStreet** [comment]
Yes, of course I also see the danger that everyone thinks their notes are important. But that could certainly be solved if, for example, others could simply remove the marking. (For me it is precisely about such blockings lasting several weeks: https://www.ls.brandenburg.de/ls/de/pressemitteilung/ansicht/~11-07-2022-l-23-sanierung-der-ernst-thaelmann-strassa-in-strausberg-geht-in-die-naechste-runde# )

**Comment by matkoniecz** [comment]
It does not really make sense to have ability to mark notes as urgent - it is additional complexity without actual benefit.

It just increases entitlement and expectations on note creator side with zero benefit.

**Comment by HolgerJeromin** [comment]
There are places in the world where people are monitoring osm notes. They will at least triage each of them and decide when to  put them into the map data. So no need for an urgent tag here.

But if there are no people monitoring an area an urgent tag will not change the situation. So no need for an urgent tag here, too.

Perhaps we have some areas in between where there are people with not much time which would monitor urgent tagged notes only. But I do not think so...

Off topic:
> For me it is precisely about such blockings lasting several weeks

Some data consumer fetches data every one or two years. Is it a good idea to remove the road for such customers?

**Comment by woodpeck** [comment]
I think that some generic form of note classification would be great - perhaps even having "tags" on notes. For example, it would be very useful to distinguish notes that "need on-location survey" from those that maybe just highlight a spelling mistake or require further research, or those requiring a re-visit at a later time, et cetera. 

Note creators might not be able to make this classification but in areas where many mappers work on the notes and "triage" them as Holger says, if one person has already looked at the note and decided it needs a survey, then 20 other people don't need to open the note too when they're not in the vicinity.

Downstream note "consumers" could then allow users to filter notes accordingly.

Such a mechanism would automatically also bring with it the capability of highlighting "important" notes.

**Comment by OpenGreenStreet** [comment]
Thank you all for your feedback.

I really like the classification thing too. Maybe it would be a compromise: a way to "nominate for a certain category of notes".
To prevent abuse, at least a second mapper would have to confirm it.
Question: for suspensions of several weeks, isn't the expected duration stored?

P.S.: By the way: this is nothing more and nothing less than a well-intentioned idea of mine.

**Comment by AntonKhorev** [comment]
> For example, it would be very useful to distinguish notes that "need on-location survey" from those that maybe just highlight a spelling mistake or require further research, or those requiring a re-visit at a later time, et cetera.

I've seen people closing notes that in their opinion need on-location survey and adding fixme tags on elements instead. I guess they'll stop doing this if there's some accepted resurvey tag.

**Comment by mnalis** [comment]
> I think that some generic form of note classification would be great - perhaps even having "tags" on notes. For example, it would be very useful to distinguish notes that "need on-location survey" from those that maybe just highlight a spelling mistake or require further research, or those requiring a re-visit at a later time, et cetera.

There is some use of text `#surveyme` inside a note used for indicating the note needs on-site survey, see e.g. https://github.com/streetcomplete/StreetComplete/issues/2641

> pbzip2 -dc planet-notes-latest.osn.bz2 | fgrep '#surveyme' | wc -l
> 2593

**Comment by pedjas** [comment]
Here is some tagging scheme we are introducing for our local use, but intention is to use it as initative to create global standard tagging.

https://community.openstreetmap.org/t/tagovi-za-odredivanje-statusa-napomena-i-komentara-napomena/4788

**Comment by mnalis** [comment]
> For example: a road has been closed due to construction. In my view, this is important information that should be added to OSM as soon as possible.

>Background:
> I contribute to OSM mostly via StreetComplete.

You may be interested to know that [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) (StreetComplete _"Expert Edition"_ fork) allows you to _**immediately**_ and directly mark the road as under construction via `Uh...` answer `It's under construction until...` on any road quest (or e.g. Surface Overlay).

That would allow you to mark such important construction works immediately; i.e. much faster than any indirect route (like _"urgent" note_), while still enjoying simplicity of StreetComplete-alike interface.

**Comment by OpenGreenStreet** [comment]
@mnalis Thank you for your tip. Is StreetComplete “Expert Edition” available via the google play store?

**Comment by OpenGreenStreet** [comment]
@mnalis Thank you, that is a great pity

**Comment by mnalis** [comment]
@OpenGreenStreet no worries, but why do you think it is a _"great pity"_?
 
Are you aware that you still **can** download and install the app on your Android phone just fine, even if it is not available on Google Play? It is just a few clicks more; it does not sound so _"horrible"_ to me as your comment seems to imply?

**Comment by mmd-osm** [comment]
@mnalis , @OpenGreenStreet , as a reminder, this is the osm website repo and not an ideal place to discuss SCEE topics. Thanks!
